compel/Makefile

```diff
@@ -17,6 +17,10 @@ ldflags-y		+= -r
 # UAPI inclusion, referred as <compel/...>
 ccflags-y		+= -I compel/include/uapi
 
+ifeq ($(ARCH),mips)
+ccflags-y		+= -I compel
+endif
+
```

Comment:
Let's have this `-I compel` for all platforms, rather than having excessive `ifeq`

Reply:
ok

Reply:
ok,I have do it